### PR TITLE
Add disclaimer that only github authentication is supported

### PR DESCRIPTION
**Pull request by michael-valdron**
### What does this PR do?:

Adds disclaimer to README that only github log in is supported for RHDH user authentication with GitLab log in planned for future versions.

**Additional Changes**

- Adds a GitLab Integration section to README to outline how to enable
- Changes 'CLI' section to 'Helm Chart Installer' to make title more clear
- Adds list of related sections for configuration section under the README

### Which issue(s) this PR fixes:

https://issues.redhat.com/browse/DEVAI-221

### PR acceptance criteria:
Testing and documentation do not need to be complete in order for this PR to be approved. We just need to ensure tracking issues are opened and linked to this PR, if they are not in the PR scope due to various constraints.

- [ ] Tested and Verified

  

- [X] Documentation (READMEs, Product Docs, Blogs, Education Modules, etc.)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # Red Hat Developer Hub Installer for AI Software Templates
 
+> We are only supporting **GitHub Authentication**, **GitLab Authentication** is planned for future versions.
```
Comment (Jdubrick) [inline README.md]: I'm wondering if this should be marked as important same as what @johnmcollier did on the template repo here https://github.com/redhat-ai-dev/ai-lab-template/commit/4ec1c75ff3cb0cd1827bb454b8a4155a534092f9?

Thoughts?

Comment (maysunfaisal) [reply README.md]: yeah sure that makes sense

Comment (maysunfaisal) [inline README.md]:
```suggestion
> Currently, only the **GitHub Authentication** is supported. **GitLab Authentication** is planned for future versions.
```

Comment (michael-valdron) [reply README.md]: addressed in https://github.com/redhat-ai-dev/ai-rhdh-installer/pull/20/commits/e41515d7a51f655a8160c0143eb35e0b273d007a

**Review by Jdubrick — APPROVED** [review]

**Comment by maysunfaisal** [comment]
@michael-valdron to add in a bit more detail about how to use `RHDH_GITLAB_INTEGRATION` and surface it a bit up front before configuration

**Review by thepetk — COMMENTED** [review]
lgtm, although I agree to the point raised here: https://github.com/redhat-ai-dev/ai-rhdh-installer/pull/20#issuecomment-2462814801

I feel it will really help the users to understand how they can possibly use this variable to use the installer through gitlab.

**Comment by michael-valdron** [comment]
>> @michael-valdron to add in a bit more detail about how to use `RHDH_GITLAB_INTEGRATION` and surface it a bit up front before configuration
>
> lgtm, although I agree to the point raised here: [#20 (comment)](https://github.com/redhat-ai-dev/ai-rhdh-installer/pull/20#issuecomment-2462814801)
> 
> I feel it will really help the users to understand how they can possibly use this variable to use the installer through gitlab.

@maysunfaisal @thepetk I have added a bit more detail in https://github.com/redhat-ai-dev/ai-rhdh-installer/pull/20/commits/0f12478f5d614b3e3d83af66475d012d451c7e35 to get someone started with GitLab integration.

**Review by thepetk — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -59,6 +62,10 @@ Detailed documentation for configuring Pipelines/Tekton can be found in [`PIPELI
 
 Detailed documentation for configuring Developer Hub can be found in [`RHDH-CONFIG.md`](./docs/RHDH-CONFIG.md).
 
+## GitLab Integration
+
+To configure RHDH to use GitLab as the git repository source, you will need to first run `export RHDH_GITLAB_INTEGRATION=true` before running any of the [configuration](#configuration) scripts. Run `export RHDH_GITHUB_INTEGRATION=false` to disable GitHub integration.
```
Comment (maysunfaisal) [inline README.md:77]: If we are going to put this section and other env config sections below; I think we can add a line before bash configure.sh to say; please scroll below to customize your configuration if you dont want the default

Comment (maysunfaisal) [reply README.md:77]: otherwise many would just run configure.sh and then wonder how to customize things..

Comment (michael-valdron) [reply README.md:77]: @maysunfaisal https://github.com/redhat-ai-dev/ai-rhdh-installer/pull/20/commits/25901016bd3d563de50dad953689dd8c4984f143 should resolve this, let me know what you think.

**Review by maysunfaisal — APPROVED** [review]

**Pull request merged**